containers/AppNavbar.tsx

```diff
@@ -14,15 +16,23 @@ export const AppNavbar: React.FC = () => {
       }
       secondary={
         <>
+          <NavbarLink href="/blog">Blog</NavbarLink>
           <NavbarLink href="https://argos-ci.com/docs/">Docs</NavbarLink>
           <NavbarLink href="/pricing">Pricing</NavbarLink>
-          <NavbarLink href="/blog">Blog</NavbarLink>
-          <NavbarLink href="https://app.argos-ci.com/login">Login</NavbarLink>
+          <NavbarLink href="https://github.com/argos-ci/argos">
+            <div className="flex gap-1 items-center">
+              <GithubLogo className="w-4 h-4 shrink-0" />
+              GitHub
+            </div>
+          </NavbarLink>
```

Comment:
I think GitHub should not be in the top navbar. It is secondary, going in GitHub is useless. So we should remove it, footer is enough.

Reply:
I understand your point, but I see the GitHub link in the header as a reflection of our identity as developers. It's more about our brand image than its actual utility. What are your thoughts?

Reply:
Then you should probably link to Organization. I think the best would be a page explicating that we are open source instead.